app/web/src/components/IdeToolbarNew/IdeToolbarNew.js

```diff
@@ -80,6 +83,63 @@ const IdeToolbarNew = ({ cadPackage }) => {
       copyTextToClipboard(window.location.href)
     }
   }
+  const PullTitleFromFirstLine = (code) => {
```

Comment:
not needed, but didn't want the `{256 hash}.stl` to be the file name, so if there's a comment as the first line it will use that for the title.